README.md

```diff
@@ -51,20 +52,16 @@ To use the newest (maybe unstable) version please add following into your compos
 }
 ```
 
-##### Using Symfony Flex
-Also it is possible to install bundle through [Symfony Flex][4]. It works for Symfony 3.3 and higher.  
-Bundle will be added to kernel file and default configuration will be created automatically.
-
-```bash
-composer config extra.symfony.allow-contrib true
-composer require eightpoints/guzzle-bundle
-```
-_Note: for symfony 3.3 and 3.4 you should install symfony/flex by yourself. From 4.0 it is included be default._
+_Note: we created [Symfony Flex Recipe][15] to speed up the installation of package._
 
 ----
 
 ## Usage
+
 ##### Load bundle in AppKernel.php:
+
+*Skip it for Symfony >= 4.0*
```

Comment:
what do you think about moving legacy (<= 4.0) to a separate README.md? So we can focus on the shiny new things in the main README.md and still provide information for older versions.

Reply:
Good idea. What do you think about link to https://github.com/8p/EightPointsGuzzleBundle/blob/v7.3.1/README.md ? Or you thought about totally another file?